App crashes on gcc compile.

The lib compiled right but when it start compilling the application it gives me that error:
```
error: linking with `gcc` failed: exit code: 1
  |
  = note: "gcc" "-Wl,--enable-long-section-names" "-fno-use-linker-plugin" "-Wl, --nxcompat" "-nostdlib" "-m64" "C:\\Users\\DoctorVWA\\.rustup\\toolchains\\stable-x86_64-pc-windows-gnu\\lib\\rustlib\\x86_64-pc-windows-gnu\\lib\\crt2.o" "C:\\Users\\DoctorVWA\\.rustup\\toolchains\\stable-x86_64-pc-windows-gnu\\lib\\rustlib\\x86_64-pc-windows-gnu\\lib\\rsbegin.o" "-L" "C:\\Users\\DoctorVWA\\.rustup\\toolchains\\stable-x86_64-pc-windows-gnu\\lib\\rustlib\\x86_64-pc-windows-gnu\\lib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\lilcord-15cb8e6d8b51a027.lilcord0.rcgu.o" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\lilcord-15cb8e6d8b51a027.lilcord1.rcgu.o" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\lilcord-15cb8e6d8b51a027.lilcord10.rcgu.o" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\lilcord-15cb8e6d8b51a027.lilcord11.rcgu.o" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\lilcord-15cb8e6d8b51a027.lilcord12.rcgu.o" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\lilcord-15cb8e6d8b51a027.lilcord13.rcgu.o" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\lilcord-15cb8e6d8b51a027.lilcord14.rcgu.o" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\lilcord-15cb8e6d8b51a027.lilcord15.rcgu.o" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\lilcord-15cb8e6d8b51a027.lilcord2.rcgu.o" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\lilcord-15cb8e6d8b51a027.lilcord3.rcgu.o" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\lilcord-15cb8e6d8b51a027.lilcord4.rcgu.o" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\lilcord-15cb8e6d8b51a027.lilcord5.rcgu.o" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\lilcord-15cb8e6d8b51a027.lilcord6.rcgu.o"
 "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\lilcord15cb8e6d8b51a027.lilcord7.rcgu.o" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\lilcord-15cb8e6d8b51a027.lilcord8.rcgu.o" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\lilcord-15cb8e6d8b51a027.lilcord9.rcgu.o" "-o" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\lilcord-15cb8e6d8b51a027.exe" "-Wl,--subsystem,windows" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\lilcord-15cb8e6d8b51a027.crate.allocator.rcgu.o" "-Wl,--gc-sections" "-nodefaultlibs" "-L" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps" "-L" "C:\\msys64\\mingw64\\lib" "-L" "C:\\msys64\\mingw64\\lib" "-L"
"C:\\msys64\\mingw64\\lib" "-L" "C:\\msys64\\mingw64\\lib" "-L" "C:\\msys64\\mingw64\\lib" "-L" "C:\\msys64\\mingw64\\lib" "-L" "C:\\msys64\\mingw64\\lib" "-L" "C:\\msys64\\mingw64\\lib" "-L" "C:\\msys64\\mingw64\\lib" "-L" "C:\\Users\\DoctorVWA\\.cargo\\registry\\src\\github.com-1ecc6299db9ec823\\winapi-x86_64-pc-windows-gnu-0.4.0\\lib" "-L" "C:\\msys64\\mingw64\\lib" "-L" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\build\\web-view-118842cd41122d28\\out" "-L" "C:\\Users\\DoctorVWA\\.rustup\\toolchains\\stable-x86_64-pc-windows-gnu\\lib\\rustlib\\x86_64-pc-windows-gnu\\lib" "-Wl,-Bstatic" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libgtk-4f6e228c353d35d0.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libgdk-fcb647845b415b3b.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libpango-483712774c188bf6.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libgdk_pixbu
f-9aa85792949717c6.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\
\deps\\libgtk_sys-7b6a2cb3d58a9155.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord
\\target\\debug\\deps\\libgdk_sys-3f75e0c9ec1fb338.rlib" "C:\\Users\\DoctorVWA\\
Desktop\\lilcord\\target\\debug\\deps\\libpango_sys-24ec8cd47f231518.rlib" "C:\\
Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libgdk_pixbuf_sys-0f254
50e1c21652e.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\
libatk_sys-42d0168c076cffbb.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\targe
t\\debug\\deps\\libgio-72895eea7e996597.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libgio_sys-55bba6f31f5b15d9.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libcairo-abb6e1489881f7d8.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libglib-6b6d3148a88811e1.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libgobject_sys-1c36f4639c197ac7.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libglib_sys-363f879bf1a50745.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\liblazy_static-4db6f52f71e2b275.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libbitflags-a0c5ccbbea449dc5.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libc_vec-66fb41a2be053426.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libcairo_sys-9c8ae188b644b7c4.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libwinapi-0aed34f6829c55ee.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\liblibc-40363f789d24d302.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\liburl-e055d48a5b753843.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libpercent_encoding-d0f5e1957f7cb514.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libidna-16faa290b01f4775.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libunicode_normalization-a2b6239c3b55ffd3.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libunicode_bidi-3dd040228b31f427.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libmatches-eba9499f8443541e.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libweb_view-40919c65876029ac.rlib" "C:\\Users\\DoctorVWA\\Desktop\\lilcord\\target\\debug\\deps\\libfnv-362faccdd1a47124.rlib" "C:\\Users\\DoctorVWA\\.rustup\\toolchains\\stable-x86_64-pc-windows-gnu\\lib\\rustlib\\x86_64-pc-windows-gnu\\lib\\libstd-a0684270ba45e4c4.rlib" "C:\\Users\\DoctorVWA\\.rustup\\toolchains\\stable-x86_64-pc-windows-gnu\\lib\\rustlib\\x86_64-pc-windows-gnu\\lib\\libpanic_unwind-9959500f08f0e1ba.rlib" "C:\\Users\\DoctorVWA\\.rustup\\toolchains\\stable-x86_64-pc-windows-gnu\\lib\\rustlib\\x86_64-pc-windows-gnu\\lib\\libunwind-15ac613a7fce0483.rlib" "C:\\Users\\DoctorVWA\\.rustup\\toolchains\\stable-x86_64-pc-windows-gnu\\lib\\rustlib\\x86_64-pc-windows-gnu\\lib\\liblibc-8a35d8972d0168d2.rlib" "C:\\Users\\DoctorVWA\\.rustup\\toolchains\\stable-x86_64-pc-windows-gnu\\lib\\rustlib\\x86_64-pc-windows-gnu\\lib\\liballoc_system-c3eb086d76a23433.rlib"  "C:\\Users\\DoctorVWA\\.rustup\\toolchains\\stable-x86_64-pc-windows-gnu\\lib\\rustlib\\x86_64-pc-windows-gnu\\lib\\liballoc-e077f5c433365928.rlib" "C:\\Users\\DoctorVWA\\.rustup\\toolchains\\stable-x86_64-pc-windows-gnu\\lib\\rustlib\\x86_64-pc-windows-gnu\\lib\\libstd_unicode-fc698978fa4eb61f.rlib" "C:\\Users\\DoctorVWA\\.rustup\\toolchains\\stable-x86_64-pc-windows-gnu\\lib\\rustlib\\x86_64-pc-windows-gnu\\lib\\libcore-732d0577648c664f.rlib" "C:\\Users\\DoctorVWA\\.rustup\\toolchains\\stable-x86_64-pc-windows-gnu\\lib\\rustlib\\x86_64-pc-windows-gnu\\lib\\libcompiler_builtins-8f1b4d5cbd48c220.rlib" "-Wl,-Bdynamic" "-l" "gtk-3" "-l" "gdk-3" "-l" "gdk-3" "-l" "pango-1.0" "-l" "gdk_pixbuf-2.0" "-l" "atk-1.0" "-l" "gio-2.0" "-l" "gobject-2.0" "-l" "glib-2.0" "-l" "gobject-2.0" "-l" "cairo" "-l" "advapi32" "-l" "ws2_32" "-l" "userenv" "-l" "shell32" "-Wl,-Bstatic" "-l" "gcc_eh" "-l" "pthread" "-Wl,-Bdynamic" "-lmingwex" "-lmingw32" "-lgcc" "-lmsvc
rt" "-luser32" "-lkernel32" "C:\\Users\\DoctorVWA\\.rustup\\toolchains\\stable-x86_64-pc-windows-gnu\\lib\\rustlib\\x86_64-pc-windows-gnu\\lib\\rsend.o"
  = note: C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o): In function `EmbedBrowserObject':
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:941: undefined reference to `__imp_CoGetClassObject'
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:964: undefined reference to `__imp_OleSetContainedObject
'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o): In function `DisplayHTMLPage':
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:1020: undefined reference to `__imp_VariantInit'
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:1028: undefined reference to `__imp_SysAllocString'
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:1040: undefined reference to `__imp_VariantClear'
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:1059: undefined reference to `__imp_SafeArrayCreate'
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:1061: undefined reference to `__imp_SafeArrayAccessData'

          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:1069: undefined reference to `__imp_SysAllocString'
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:1080: undefined reference to `__imp_SafeArrayDestroy'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o): In function `webview_init':
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:1173: undefined reference to `__imp_OleInitialize'
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:1207: undefined reference to `__imp_OleUninitialize'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o): In function `webview_eval':
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:1285: undefined reference to `__imp_SysAllocString'
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:1289: undefined reference to `__imp_SysFreeString'
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:1292: undefined reference to `__imp_SysFreeString'
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:1312: undefined reference to `__imp_SysAllocString'
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:1318: undefined reference to `__imp_SysFreeString'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o): In function `webview_dialog':
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:1471: undefined reference to `__imp_CoCreateInstance'
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:1484: undefined reference to `__imp_CoCreateInstance'
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:1514: undefined reference to `__imp_CoTaskMemFree'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o): In function `webview_exit':
          C:\Users\DoctorVWA\.cargo\git\checkouts\web-view-9c9481cc535a4781\310f26a/webview-c/webview.h:1547: undefined reference to `__imp_OleUninitialize'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o):lib.c:(.rdata$.refptr.CLSID_FileSaveDialog[.refptr.CLSID_FileSaveDialog]+0x0): undefined reference to `CLSID_FileSaveDialog'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o):lib.c:(.rdata$.refptr.IID_IFileSaveDialog[.refptr.IID_IFileSaveDialog]+0x0): undefined reference to `IID_IFileSaveDialog'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o):lib.c:(.rdata$.refptr.CLSID_FileOpenDialog[.refptr.CLSID_FileOpenDialog]+0x0): undefined reference to `CLSID_FileOpenDialog'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o):lib.c:(.rdata$.refptr.IID_IFileOpenDialog[.refptr.IID_IFileOpenDialog]+0x0): undefined reference to `IID_IFileOpenDialog'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o):lib.c:(.rdata$.refptr.GUID_NULL[.refptr.GUID_NULL]+0x0): undefined reference to `GUID_NULL'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o):lib.c:(.rdata$.refptr.IID_IOleInPlaceActiveObject[.refptr.IID_IOleInPlaceActiveObject]+0x0): undefined reference to `IID_IOleInPlaceActiveObject'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o):lib.c:(.rdata$.refptr.IID_IHTMLDocument2[.refptr.IID_IHTMLDocument2]+0x0): undefined reference to `IID_IHTMLDocument2'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o):lib.c:(.rdata$.refptr.IID_IWebBrowser2[.refptr.IID_IWebBrowser2]+0x0): undefined reference to `IID_IWebBrowser2'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o):lib.c:(.rdata$.refptr.IID_IOleObject[.refptr.IID_IOleObject]+0x0): undefined reference to `IID_IOleObject'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o):lib.c:(.rdata$.refptr.CLSID_WebBrowser[.refptr.CLSID_WebBrowser]+0x0): undefined reference to `CLSID_WebBrowser'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o):lib.c:(.rdata$.refptr.IID_IClassFactory[.refptr.IID_IClassFactory]+0x0): undefined reference to `IID_IClassFactory'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o):lib.c:(.rdata$.refptr.IID_IOleInPlaceObject[.refptr.IID_IOleInPlaceObject]+0x0): undefined reference to `IID_IOleInPlaceObject'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o):lib.c:(.rdata$.refptr.IID_IDocHostUIHandler[.refptr.IID_IDocHostUIHandler]+0x0): undefined reference to `IID_IDocHostUIHandler'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o):lib.c:(.rdata$.refptr.IID_IOleInPlaceSite[.refptr.IID_IOleInPlaceSite]+0x0): undefined reference to `IID_IOleInPlaceSite'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o):lib.c:(.rdata$.refptr.IID_IOleClientSite[.refptr.IID_IOleClientSite]+0x0): undefined reference to `IID_IOleClientSite'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o):lib.c:(.rdata$.refptr.IID_IUnknown[.refptr.IID_IUnknown]+0x0): undefined reference to `IID_IUnknown'
          C:\Users\DoctorVWA\Desktop\lilcord\target\debug\deps\libweb_view-40919c65876029ac.rlib(lib.o):lib.c:(.rdata$.refptr.IID_IDispatch[.refptr.IID_IDispatch]+0x0): undefined reference to `IID_IDispatch'
```

--- a/build.rs
+++ b/build.rs
@@ -15,6 +15,9 @@ fn main() {
 	let target = env::var("TARGET").unwrap();
 	if target.contains("windows") {
 		build.define("WEBVIEW_WINAPI", None);
+		for &lib in &["ole32", "comctl32", "oleaut32", "uuid"] {
+			println!("cargo:rustc-link-lib={}", lib);
+		}
 	} else if target.contains("linux") || target.contains("bsd") {
 		let webkit = pkg_config::Config::new().atleast_version("2.8").probe("webkit2gtk-4.0").unwrap();
 

